src/transformStateWithClones.js

```diff
@@ -7,7 +7,41 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  let currentState = { ...state };
+
+  for (const action of actions) {
+    let nextState = { ...currentState };
```

Comment:
There is actually no need to make a copy one more time

Reply:
do you mean I shouldnt write this part:
    let nextState = { ...currentState }; 
?

If yes, how can I rewrite it?